pom.xml

```diff
@@ -248,6 +248,11 @@
                 <artifactId>junit</artifactId>
                 <version>${dependency.junit.version}</version>
             </dependency>
+            <dependency>
+                <groupId>org.yaml</groupId>
+                <artifactId>snakeyaml</artifactId>
```

Comment:
Why we need this dependency?

Reply:
During the development of this PR, Veracode triggered a problem with that dependency, which didn't allowed to proceed the Travis build.

Meanwhile, I've implemented the same change on master, using [ACS-3492](https://alfresco.atlassian.net/browse/ACS-3492).

Reply:
Where we get this dependency? If we already have snakeyaml in the transform-core then we should just upgrade it.